src/services/index.ts

```diff
@@ -6,6 +6,11 @@ const backendUrl = url => {
   return `${process.env.BACKEND_URL}${url}`;
 };
 
+export const getSushiPool = async (address: String) => {
+  const res = await agent.get<any>('https://external-services.azurewebsites.net/api/sushi').query({ address });
```

Comment:
Make this a configuration parameter. Hard coded addresses are annoying to keep track of